exec/java-exec/src/main/resources/rest/index.ftl

```diff
@@ -112,6 +112,14 @@
                         <td>Admin User Groups</td>
                         <td class="list-value">${model.getAdminUserGroups()}</td>
                       </tr>
+                      <tr>
+                        <td>Process User</td>
+                        <td class="list-value">${model.getProcessUser()}</td>
```

Comment:
Please look at the generated HTML to ensure that the angle brackets in "<none>" are escaped. Should happen [by default](http://freemarker.org/docs/dgui_misc_autoescaping.html) if we've set up Freemarker correctly.

Reply:
yes freemarker handles this